vscode/.vscodeignore

```diff
@@ -2,3 +2,4 @@ src/**
 .gitignore
 tsconfig.json
 **/*.map
+../**/*
```

Comment:
Why is this here? You're one directory down (in /vscode), and you're telling it to go up a level and ignore all subdirectories. So.... ignore ALL files in the repo then?

Reply:
Without this change the `vsce package` was telling that thousands of files will be packed. The `vsce ls` was showing that in addition to the contents of the current dir the whole contents of the upper dir (`../`) will be packed. 

After this change only the files in the current dir were shown as the ones to be backed and `vsce package` did not warn about thousands of files.